# Owned supplemental data + research frontier
**Employer-facing, semi-reproducible aggregate notebook**

This notebook summarizes the post-PR-34 research program using public-safe aggregate tables committed with the repository. It does **not** contain private predictions, source-response archives, fitted production models, exact residual gates/weights, credentials, or AWS paths.

All 2026 scores shown here are post-competition research measurements, not original leaderboard claims.

In [1]:
from pathlib import Path
import pandas as pd
import plotly.graph_objects as go
import plotly.express as px

root = Path.cwd()
portfolio = root if root.name == 'portfolio' else (root / 'portfolio' if (root / 'portfolio').is_dir() else root)
exp = pd.read_csv(portfolio / 'post_merge_experiments.csv')
ownership = pd.read_csv(portfolio / 'supplemental_source_ownership.csv')
canonical = 0.1033437336
winner = 0.1097454
target = 0.09
pending = 0.1027974461
print({'canonical_aws_brier': canonical, 'published_2026_winner': winner, 'stretch_target': target, 'pending_local_candidate': pending, 'experiment_rows': len(exp)})

{'canonical_aws_brier': 0.1033437336, 'published_2026_winner': 0.1097454, 'stretch_target': 0.09, 'pending_local_candidate': 0.1027974461, 'experiment_rows': 23}


## 1. Score progression
Lower Brier is better. The **0.1033437** point is the current AWS-reproduced research boundary. The **0.1027974** point is deliberately shown as pending rather than promoted because canonical AWS reproduction has not yet occurred.

In [2]:
score = pd.DataFrame({
    'boundary':['Published winner','Prior retained','External consensus','AWS current','Pending local','Stretch target'],
    'brier':[0.1097454,0.1051853186,0.1043512820,canonical,pending,target],
})
fig = go.Figure(go.Scatter(x=score.boundary, y=score.brier, mode='lines+markers+text', text=[f'{x:.7f}' for x in score.brier], textposition='top center'))
fig.update_layout(title='Post-competition research score progression', yaxis_title='Brier (lower is better)', yaxis_range=[0.088,0.112], height=460)
fig.show()

Post-competition research score progression

## 2. Owned supplemental source program
The table below reports **pipeline ownership/provenance state**, not ownership of third-party intellectual property. 'Owned' means the private pipeline can independently acquire/normalize the source or reconstruct a separately named analogue without depending on another competitor's prepared feature CSV.

In [3]:
state = ownership['state'].value_counts().rename_axis('state').reset_index(name='count')
fig = px.bar(state, x='state', y='count', text='count', title='Supplemental source ownership/provenance states')
fig.update_layout(height=430, xaxis_tickangle=-25)
fig.show()
display(ownership)

Supplemental source ownership/provenance states

## 3. Experiment discipline
The post-release program separates engineering failures, source/timing blocks, and scientific rejections. The public table focuses on scientific milestone state; private operational ledgers remain outside the repository.

In [4]:
status = exp['status'].value_counts().rename_axis('status').reset_index(name='count')
fig = px.bar(status, x='status', y='count', text='count', title='Published post-merge milestone states')
fig.update_layout(height=430, xaxis_tickangle=-25)
fig.show()

Published post-merge milestone states

In [5]:
selected = exp[exp['milestone'].isin([
    'Owned preseason trajectory plus coach history','Owned player-value availability v1','AP first-place style correction v12','Sparse pruned LR residual v15','Robust rating consensus v16','Uncertainty gated Torvik v18'
])].copy()
fig = go.Figure()
fig.add_trace(go.Bar(name='Development', x=selected.milestone, y=selected.development_gain))
fig.add_trace(go.Bar(name='Recent confirmation', x=selected.milestone, y=selected.recent_confirmation_gain))
fig.update_layout(title='Selected transfer tests: development vs recent confirmation', barmode='group', yaxis_title='Brier improvement', height=500, xaxis_tickangle=-25)
fig.show()

Selected transfer tests

## 4. Current interpretation
- The strongest verified gains come from **selective residual correction**, not wholesale replacement of the retained system.
- Source reconstruction can be a success even when its first model use is rejected.
- Historical source integrity and historical point-in-time publication proof are different standards.
- Active modeling scope is **men 2003+ / women 2010+**; earlier cached material is not an active research target.
- The 2027 pipeline prioritizes prospective timestamps, checksums, waiting states, and source eligibility before modeling.

The next public promotion should occur only after the pending candidate is reproduced canonically or a later prospectively frozen model establishes a stronger boundary.

In [6]:
assert abs(exp.loc[exp['canonical'].fillna(False), 'audit_brier'].dropna().iloc[-1] - canonical) < 1e-10
assert {'AP Poll','Dated BartTorvik','WNCAA NET'}.issubset(set(ownership['source']))
print('PUBLIC_OWNED_DATA_FRONTIER_COMPLETE — aggregate evidence only; private forecasting implementation intentionally excluded')

PUBLIC_OWNED_DATA_FRONTIER_COMPLETE — aggregate evidence only; private forecasting implementation intentionally excluded
